components/common/src/ome/api/Search.java

```diff
@@ -158,7 +158,7 @@
 
     /**
      * Returns the current case sensitivity setting. If
-     * {@link #setCaseSentivice()} has not been called, the
+     * {@link #setCaseSentivice(boolean)} has not been called, the
```

Comment:
A followup PR would be fine.  Could be deferred to 5.2 if it's going to break stuff, but if it's already broken then it's probably safe to fix now.